src/js/app/utils/createItem.js

```diff
@@ -264,7 +264,6 @@ export const createItem = (origin = null, serverFileReference = null, file = nul
             state.activeProcessor = null;
 
             // if file was uploaded but processing was cancelled during perceived processor time store file reference
-            state.transferId = null;
```

Comment:
shouldn't this be kept?

Reply:
I don't actually know why the transfer id has to be cleared.
But I do know that I need to keep the id for the revert which is going to happen next.